bentoml/configuration/default_bentoml.cfg

```diff
@@ -99,5 +99,6 @@ bento_uri_default_expiration = 3000
 [ngsild]
 cb_url = https://stellio-dev.eglobalmark.com
 at_context = https://raw.githubusercontent.com/easy-global-market/ngsild-api-data-models/master/mlaas/jsonld-contexts/mlaas-precipitation-contexts.jsonld
-access_token = changeme
-ml_model_id=urn:ngsi-ld:MLModel:flow:predict
+access_token = MY_TOKEN_ID
+ml_model_id = urn:ngsi-ld:MLModel:flow:predict
+ml_model_input = precipitation
```

Comment:
I guess you could also have the `ml_model_output` here

Reply:
Possibly. Should I add it ?

Reply:
Cleaner if both are extracted in conf.